src/focus.service.ts

```diff
@@ -509,6 +509,11 @@ export class FocusService {
   private isFocusable(el: HTMLElement) {
     const role = el.getAttribute('role');
     const tabIndex = el.tabIndex;
+    const visible = window.getComputedStyle(el).display !== 'none';
```

Comment:
If you're going this route you can leverage [`el.offsetParent`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/offsetParent), which will be null for hidden elements that are not fixed-positioned